lib/module.js

```diff
@@ -170,7 +166,13 @@ Module._findPath = function(request, paths) {
 
     if (filename) {
       // Warn once if '.' resolved outside the module dir
-      if (request === '.' && i > 0) noopDeprecateRequireDot();
+      if (request === '.' && i > 0) {
+        warned = internalUtil.printDeprecationMessage(
+          '(node) warning: require(\'.\') resolved outside the package ' +
+          'directory. This functionality is deprecated and will be removed ' +
+          'soon.', warned);
+      }
```

Comment:
I think we are going circles here with `(node)` included in the string. Is it not possible to just let `internalUtil.printDeprecationMessage` append it?